main.cpp

```diff
@@ -215,6 +260,51 @@ Thing 2) Dentist
     3) pay staff wages
  */
 
+struct Dentist 
+{
+    
+    //number of patients per day
+    int numPatients = 8;
+    //maximum number of patients per week
+    int maxPatientsPerWeek = 40;
+    //disposable equipment costs per week
+    double costDisposableEquipment = 2000;
+    //profit per week (float)
+    double profitPerWeek = 20000;
+    //number of staff (int)
+    int numStaff = 20;
+
+    struct Person
+    {
+    //personName
+    std::string personName = "Mary";
+    //person age in years
+    int personAge = 35;
+    //person gender
+    std::string personGender = "female";
+    //personInsuranceCompany 
+    std::string personInsuranceCompany = "No Insurance";
+    //date of last check (ddmmyyyy)
+    int dateOfLastCheck = 01112022;
+
+    //return patient's insurance company
+    std::string getInsuranceCompany(std::string personInsuranceCompany); //returns name of patient's insurance company
+    //dental treatment deemed necessary
+    int dateOfNextDentalAppointment(int dateOfLastCheck); //returns month when next check will be
+    //cost of work undertaken
+    float costOfDentalTreatment(); //returns cost of treatment
+    };
```

Comment:
Fix this indentation
Code inside of curly braces needs to be indented four spaces from those curly braces
You do not need to include the word `person` in any of these member variable names.  The class name already tells us that it is a person
- [ ] revise